global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.1.100"
+  }
+}
```

Comment:
Let's have a newline at the end of files.

Reply:
Even `dotnet` won't add them, but okay.